src/problem_depot/problems/sdp.jl

```diff
@@ -1361,3 +1361,35 @@ end
         end
     end
 end
+
+@add_problem sdp function sdp_min_maxeig_canon_lmi(handle_problem!, ::Val{test}, atol, rtol, ::Type{T}) where {T, test}
+    # Minimize the maximum eigenvalue of an affine matrix function of a vector argument. Formulated
+    # as a linear optimization subject to a canonical LMI (SDP) constraint:
+    #
+    # minimize      λ
+    # subject to    A(x)⪯λ
+    # where x is a vector in Rⁿ and the matrix function A(x) = A₀ + A₁x₁ + A₂x₂ + … + Aₙxₙ.
+    #
+    # Besides serving as a benchmark, it also tests the solution of the Issue
+    # https://github.com/jump-dev/Convex.jl/issues/447
+    # (the issues boils down to  evaluation of `A₁x₁`, because `x₁` was a matrix of size (1,1)).
+
+    A₀ = [1.0 2.0; 2.0 3.0]
+    A₁ = [1.0 0.0; 0.0 -1.0]
+    A₂ = [4.0 5.0; 5.0 -6.0]
+
+    x = Variable(2)
+    λ = Variable()
+
+    A = A₀ + A₁*x[1] + A₂*x[2]
+
+    p = minimize(λ,A⪯λ)
```

Comment:
This allows the problem to be used with different types (eg some solvers can handle BigFloats)
```suggestion
    p = minimize(λ,A⪯λ; numeric_type=T)
```